<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="utf-8">
    <title>Bioinformatics Blogs</title>
    <link rel="stylesheet" href="style.css">
    <!-- RSS autodiscovery -->
    <link rel="alternate" type="application/rss+xml" title="Bioinformatics Blogs RSS" href="feed.xml" />
</head>

<body>
    <header>
        <h1>Bioinformatics Blogs</h1>
        <p>A feed of the latest posts of Bioinformatics blogs around the world</p>
        <p><a href="feed.xml">üì° Subscribe via RSS</a></p>
    </header>
    <main>
        <div id="content">
            
        <article>
          <h2><a href="https://omicsomics.blogspot.com/2025/06/could-it-have-been-found-with-short.html">Could It Have Been Found With Short Reads?</a></h2>
          <p class="meta">From <strong>Omics! Omics!</strong> ‚Äì 2025-06-29T22:56:00.001-04:00</p>
          <p><div>Initially, the ESHG program was overwhelming.¬† With the exception of the official opening and closing sessions, every timeslot had multiple parallel sessions - and sometimes those also had compet...</p>
        </article>
        

        <article>
          <h2><a href="https://ekernf01.github.io/virtual-cell-june-2025">A recap of virtual cell releases circa June 2025</a></h2>
          <p class="meta">From <strong>Eric Kernfeld</strong> ‚Äì 2025-06-29T00:00:00+00:00</p>
          <p>In October 2024, I twote that ‚Äúsomething is deeply wrong‚Äù with what we now call virtual cell models. A lot has happened since then: modelers are advancing new architectures and mining new sources of i...</p>
        </article>
        

        <article>
          <h2><a href="https://omicsomics.blogspot.com/2025/06/food-for-thought-on-onts-proteomics-push.html">Food For Thought On ONT's Proteomics Push</a></h2>
          <p class="meta">From <strong>Omics! Omics!</strong> ‚Äì 2025-06-26T08:13:00.001-04:00</p>
          <p><div>This year, instead of touring London after the Nanopore confab I headed to Italy for the European Society of Human Genetics meeting.¬† Upon hearing Lakmal Jayasinghe describe ONT's proteomics plan...</p>
        </article>
        

        <article>
          <h2><a href="https://ekernf01.github.io/vanilla">Five flavors of stratified LD score regression (Part 1, Vanilla)</a></h2>
          <p class="meta">From <strong>Eric Kernfeld</strong> ‚Äì 2025-06-22T00:00:00+00:00</p>
          <p>Linkage disequilibrium score regression (LDSC) is a workhorse technique at the intersection of quantitative genetics and functional genomics. The core use of S-LDSC is to detect and quantify functiona...</p>
        </article>
        

        <article>
          <h2><a href="https://ekernf01.github.io/strawberry">Five flavors of stratified LD score regression (Part 2, Strawberry)</a></h2>
          <p class="meta">From <strong>Eric Kernfeld</strong> ‚Äì 2025-06-22T00:00:00+00:00</p>
          <p>Stratified linkage disequilibrium score regression (S-LDSC) is a workhorse technique at the intersection of human genetics and functional genomics, but its exposition is heavily genetics-coded. In cel...</p>
        </article>
        

        <article>
          <h2><a href="https://ekernf01.github.io/cranberry">Five flavors of stratified LD score regression (Part 3, Cranberry)</a></h2>
          <p class="meta">From <strong>Eric Kernfeld</strong> ‚Äì 2025-06-22T00:00:00+00:00</p>
          <p>Stratified linkage disequilibrium score regression (S-LDSC) is a workhorse technique at the intersection of human genetics and functional genomics, but its exposition is heavily genetics-coded. In cel...</p>
        </article>
        

        <article>
          <h2><a href="https://omicsomics.blogspot.com/2025/06/roche-gives-sbx-updates-and-name.html">Roche Gives SBX Updates - and a Name!</a></h2>
          <p class="meta">From <strong>Omics! Omics!</strong> ‚Äì 2025-06-02T08:35:00.000-04:00</p>
          <p>Last week I double-dipped on conferences, going from London Calling to European Society for Human Genetics (ESHG) in Milan.¬† I have a raft of notes and ideas from these, which I'll try to spool out ov...</p>
        </article>
        

        <article>
          <h2><a href="https://rrwick.github.io/2025/05/27/dorado-v1.html">Dorado v1.0.0 and the v5.2.0 basecalling models</a></h2>
          <p class="meta">From <strong>Ryan Wick‚Äôs bioinformatics blog</strong> ‚Äì 2025-05-27T00:00:00+00:00</p>
          <p>...</p>
        </article>
        

        <article>
          <h2><a href="https://omicsomics.blogspot.com/2025/05/oxford-nanopore-should-spin-out-protein.html">Oxford Nanopore Should Spin Out Protein Sequencing</a></h2>
          <p class="meta">From <strong>Omics! Omics!</strong> ‚Äì 2025-05-21T02:46:00.001-04:00</p>
          <p><div>I've toyed with writing something on these lines for a long time but never quite pulled the trigger.¬† But the more I think about it, the more imperative my logic feels for spinning off the nascen...</p>
        </article>
        

        <article>
          <h2><a href="http://jefworks-lab.github.io//blog/2025/04/22/harmony-with-sketching-in-seurat/">Multi-sample Integrative Analysis of Spatial Transcriptomics Data using Sketching and Harmony in Seurat</a></h2>
          <p class="meta">From <strong>JEFworks Lab</strong> ‚Äì 2025-04-22T00:00:00+00:00</p>
          <p>I often write up code tutorials as a way for me to take notes on what I‚Äôve been learning. But they also serve a secondary purpose as training material for students in my lab. In particular, they provi...</p>
        </article>
        

        <article>
          <h2><a href="https://rrwick.github.io/2025/02/19/fastq-assemblies.html">FASTQ assemblies with Dorado polish</a></h2>
          <p class="meta">From <strong>Ryan Wick‚Äôs bioinformatics blog</strong> ‚Äì 2025-02-19T00:00:00+00:00</p>
          <p>...</p>
        </article>
        

        <article>
          <h2><a href="https://rrwick.github.io/2025/02/07/dorado-polish.html">Medaka vs Dorado polish</a></h2>
          <p class="meta">From <strong>Ryan Wick‚Äôs bioinformatics blog</strong> ‚Äì 2025-02-07T00:00:00+00:00</p>
          <p>...</p>
        </article>
        

        <article>
          <h2><a href="http://ivory.idyll.org/blog/2024-announcing-chill-filter.html">Announcing: the chill-filter Web site - DNA sample composition at a glance!</a></h2>
          <p class="meta">From <strong>Living in an Ivory Basement</strong> ‚Äì 2024-12-27T00:00:00+01:00</p>
          <p><p>what's in my sample, dude?</p>...</p>
        </article>
        

        <article>
          <h2><a href="https://rrwick.github.io/2024/12/17/cycloneseq.html">A first look at CycloneSEQ data</a></h2>
          <p class="meta">From <strong>Ryan Wick‚Äôs bioinformatics blog</strong> ‚Äì 2024-12-17T00:00:00+00:00</p>
          <p>...</p>
        </article>
        

        <article>
          <h2><a href="http://jefworks-lab.github.io//blog/2024/11/04/using-ai-to-prevent-manels/">Using AI to find heterogeneous scientific speakers</a></h2>
          <p class="meta">From <strong>JEFworks Lab</strong> ‚Äì 2024-11-04T00:00:00+00:00</p>
          <p>(Alt title: how I am using AI to prevent manels and how you can too)...</p>
        </article>
        

        <article>
          <h2><a href="http://ivory.idyll.org/blog/2024-why-i-like-rust.html">Why Rust is an increasingly beloved part of my programming toolbox</a></h2>
          <p class="meta">From <strong>Living in an Ivory Basement</strong> ‚Äì 2024-10-16T00:00:00+02:00</p>
          <p><p>Rust is nice!</p>...</p>
        </article>
        

        <article>
          <h2><a href="http://jefworks-lab.github.io//blog/2024/08/29/the-many-ways-to-detect-svgs-using-moransI/">The many ways to calculate Moran‚Äôs I for identifying spatially variable genes in spatial transcriptomics data</a></h2>
          <p class="meta">From <strong>JEFworks Lab</strong> ‚Äì 2024-08-29T00:00:00+00:00</p>
          <p>In this blog post, I will compare 4 implementations of Moran‚Äôs I in R (MERINGUE, Rfast2, ape, and spdep) to identify spatially variable genes (SVGs) in spatial transcriptomics data....</p>
        </article>
        

        <article>
          <h2><a href="http://ivory.idyll.org/blog/2024-sourmash-advances.html">Recent advances in the sourmash ecosystem (August 2024)</a></h2>
          <p class="meta">From <strong>Living in an Ivory Basement</strong> ‚Äì 2024-08-16T00:00:00+02:00</p>
          <p><p>sourmash is faster and better than before!</p>...</p>
        </article>
        

        <article>
          <h2><a href="http://jefworks-lab.github.io//blog/2024/07/23/spatial-bootstrapping-with-seraster/">Characterizing spatial heterogeneity using spatial bootstrapping with SEraster</a></h2>
          <p class="meta">From <strong>JEFworks Lab</strong> ‚Äì 2024-07-23T00:00:00+00:00</p>
          <p>SEraster aggregates single cell spatial omics data into square/hex pixels for scalable analysis...</p>
        </article>
        

        <article>
          <h2><a href="http://ivory.idyll.org/blog/2024-speeding-sourmash-the-heck-up.html">Speeding sourmash the heck up</a></h2>
          <p class="meta">From <strong>Living in an Ivory Basement</strong> ‚Äì 2024-02-20T00:00:00+01:00</p>
          <p><p>Faster things are always nice, right?</p>...</p>
        </article>
        

            <!-- Pagination placeholder -->
            <div class="pagination">
                <a href="index.html">‚Üê Previous</a>
            </div>
        </div>
        <aside>
            <h3>Feeds</h3>
            <ul>
                <li><a href="https://davetang.org/muse/">Dave Tang's blog</a></li>
<li><a href="http://ivory.idyll.org/blog/">Living in an Ivory Basement</a></li>
<li><a href="https://ekernf01.github.io/">Eric Kernfeld</a></li>
<li><a href="http://jefworks-lab.github.io//">JEFworks Lab</a></li>
<li><a href="https://joynyaanga.com/">Joy McIlvaine, PhD</a></li>
<li><a href="https://divingintogeneticsandgenomics.com/">Chatomics on Chatomics</a></li>
<li><a href="https://rrwick.github.io/">Ryan Wick‚Äôs bioinformatics blog</a></li>
<li><a href="https://omicsomics.blogspot.com/">Omics! Omics!</a></li>
            </ul>
        </aside>
    </main>
</body>

</html>